02week/pigLatin.js

```diff
@@ -9,9 +9,27 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
-
-  // Your code here
-
+  word = word.toLowerCase().trim();
+  let vowels = ['a','i','o','e','u'];
```

Comment:
I like the loops, and the concise code, but it does handle the words with no vowels gracefully